Document DBM > Databases (#20430)
DataDog/documentation: content/en/database_monitoring/database_hosts.md

---
title: Exploring Database Hosts
kind: documentation
description: Explore and dig into your database host health and configuration

---

{{< site-region region="gov" >}}
<div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
{{< /site-region >}}

> On the [Databases page][1], you can assess the health and activity of your database hosts. Sort and filter the list to prioritize hosts with triggered alerts, high query volume, and other criteria.

Reviewer: ```suggestion
Assess the health and activity of your database hosts. Sort and filter the list to prioritize hosts with triggered alerts, high query volume, and other criteria.
```

> {{< img src="database_monitoring/databases-list.png" alt="The Databases page in Datadog" style="width:90%;" >}}

Reviewer: I don't think this image adds value here, it can be confusing since the direction to click is sandwiched between two images. Optional, recommend removing.

> Click on any host in the list to open a details panel:

Reviewer: ```suggestion
On the [Databases page][1], click on any host in the list to open a details panel:
```

|                                                 | Postgres  | SQL Server | MySQL     | Oracle    |
|-------------------------------------------------|-----------|------------|-----------|-----------|
| [Top queries](#top-queries)                     | {{< X >}} | {{< X >}}  | {{< X >}} | {{< X >}} |
| [Metrics](#metrics)                             | {{< X >}} | {{< X >}}  |           |           |
| [Active connections](#active-connections)       | {{< X >}} | {{< X >}}  | {{< X >}} | {{< X >}} |
| [Blocking queries](#blocking-queries)           | {{< X >}} | {{< X >}}  |           |           |
| [Calling services](#calling-services)           | {{< X >}} | {{< X >}}  | {{< X >}} |           |
| [Configuration details](#configuration-details) | {{< X >}} | {{< X >}}  | {{< X >}} |           |

## Top queries

On the **Top Queries** tab of the host details panel, you can sort the most common queries by maximum duration, average latency, and more.

{{< img src="database_monitoring/db-list-top-queries.png" alt="The Top Queries tab of the details panel for a single database host on the Databases page" style="width:90%;" >}}

> Click on any query to open a details panel that includes

Reviewer: ```suggestion
Click on any query statement to open a details panel that includes:
```

- query insights
- graphs for average latency and other key metrics
- explain plans
- blocking activity
- hosts that have run the query
- calling services

{{< img src="database_monitoring/db-list-query-details.png" alt="The details panel for an individual top query" style="width:90%;" >}}

## Metrics

On the **Metrics** tab of the host details panel, you can view and filter metrics for system health, query activity, blocking operations, function performance, and other key areas.

{{< img src="database_monitoring/db-list-metrics.png" alt="The Metrics tab of the details panel for a single database host on the Databases page" style="width:90%;" >}}

## Active connections

The **Active Connections** tab of the host details panel displays the live queries being executed on the host.

{{< img src="database_monitoring/db-list-active-connections.png" alt="The Active Connections tab of the details panel for a single database host on the Databases page" style="width:90%;" >}}

> Click on an individual query to open a panel that includes event attributes, related traces, and other relevant details.

Reviewer: ```suggestion
Click on a query statement to open a panel that includes event attributes, related traces, and other relevant details.
```

- the number of waiting queries

You can search and filter the queries or samples. Click any individual query row to view details.

{{< img src="database_monitoring/db-list-blocking-queries.png" alt="The Blocking Queries tab of the details panel for a single database host on the Databases page" style="width:90%;" >}}

## Calling services

On the **Calling Services** tab of the host details panel, you can view the list of services that have called the host. The displayed service information includes when the service was deployed, the number of requests made to the host per second, how many database queries were executed, and more.

{{< img src="database_monitoring/db-list-calling-services.png" alt="The Calling Services tab of the details panel for a single database host on the Databases page" style="width:90%;" >}}

Click any service row to view its APM dashboard.

## Configuration details

<div class="alert alert-info">The host must have <code>collect_settings</code> enabled in its <a href="https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/data/conf.yaml.example#L397">instance configuration</a> for this feature to work properly.</div>

> The **Configuration** tab of the host details panel provides a direct view into the host's configuration parameters without compromising database security. Use it to identify misconfigured database parameters and fine-tune settings to optimize database performance.

Reviewer: ```suggestion
The **Configuration** tab of the host details panel provides a direct view into the host's configuration parameters without compromising database security. Use it to identify misconfigured database parameters and adjust settings to optimize database performance.
```
